modules/cloud-init/variables.tf

```diff
@@ -0,0 +1,74 @@
+variable "airgap_package_url" {
+  default     = ""
+  description = "The URL of an airgap package which contains a TFE release."
+  type        = string
+}
+
+variable "airgap_installer_url" {
+  default     = "https://install.terraform.io/installer/replicated-v5.tar.gz"
+  description = "The URL of an airgap package which contains the cluster installer."
+  type        = string
+}
+
+variable "application_config" {
+  description = "The application configuration."
+  type        = map(map(string))
+}
+
+variable "custom_ca_cert_url" {
+  default     = ""
+  description = "The URL of a certificate authority bundle which contains custom certificates to be trusted by the application."
+  type        = string
+}
+
+variable "distribution" {
+  default     = "ubuntu"
+  description = "The type of Linux distribution which will be running on the machines."
+  type        = string
+}
+
+variable "license_file" {
+  description = "The pathname of a Replicated license file for the application."
+  type        = string
+}
+
+variable "proxy_address" {
+  description = "The IP address of the Kubernetes API."
+  type        = string
+}
+
+variable "proxy_url" {
+  default     = ""
+  description = "The URL of a proxy through which application traffic will be routed."
+  type        = string
+}
+
+variable "ptfe_url" {
+  default     = "https://install.terraform.io/installer/ptfe-0.1.zip"
```

Comment:
For now this should be fine, but we def want to look at how we do this moving forward